content/development-guide/api/task-scheduler.md

```diff
@@ -24,9 +23,9 @@ namespace SamplePlugin
 	{
 		private ITaskScheduler taskScheduler;
 
-		public Main (IDependencyContainer container, ITaskScheduler taskScheduler) : base ("Sample Plugin", container)
+		public Main (IDependencyContainer container, ITaskScheduler _taskScheduler) : base ("Sample Plugin", container)
 		{
-			this.taskScheduler = taskScheduler;
+			taskScheduler = _taskScheduler;
```

Comment:
???

Reply:
Our codestyle does not permit variables with underscore, especially not for method parameters!
Method parameters, local variables and private fields are written with camel case.